unit_tests/bg/citations/ref/margaret_ann_craney_d_1909.json

```diff
@@ -1,14 +1,14 @@
 {
   "std_inline": {
-    "citation": "<ref>\n'''Memorial''':\n\"Billion Graves\"<br/>\nNewcastle upon Tyne (Saint Nicholas) Cemetery, Westgate Road, Newcastle upon Tyne, England, United Kingdom<br/>\n[https://billiongraves.com/grave/Margaret-Ann-Craney/27550409 BillionGraves memorial] (accessed 6 May 2021)<br/>\nMemorial page for Margaret Ann Craney (d August 1909); Transcribed by DdraigGoch, June 1, 2018; Photographed by Lindsay, May 25, 2018.<br/>Also on memorial : Hugh Craney d 1951.\n</ref>",
+    "citation": "<ref>\n'''Memorial''':\n\"Billion Graves\"<br/>\nNewcastle upon Tyne (Saint Nicholas) Cemetery, Westgate Road, Newcastle upon Tyne, Tyne and Wear, United Kingdom<br/>\n[https://billiongraves.com/grave/Margaret-Ann-Craney/27550409 BillionGraves memorial] (accessed 6 May 2021)<br/>\nMemorial page for Margaret Ann Craney (d August 1909); Transcribed by DdraigGoch, June 1, 2018; Photographed by Lindsay, May 25, 2018.<br/>Also on memorial : Hugh Craney d 1951.\n</ref>",
```

Comment:
It seems like it is replacing the country with the county. Is that intentional?

Reply:
Fixed with latest update